repos/fdbt-site/tests/pages/api/pricingPerDistance.test.ts

```diff
@@ -337,7 +337,7 @@ describe('validate input tests', () => {
             },
         ];
         const errorsResult: ErrorInfo[] = [
-            { id: 'distance-to-0', errorMessage: 'Distances must be numbers to 2 decimal places' },
```

Comment:
Doesnt appear we have a test showing that it refuses decimal inputs - please could we add one?